pilot/helpers/ipc.py

```diff
@@ -28,8 +28,17 @@ def listen(self):
             return
 
         while True:
-            data = self.client.recv(65536)
-            message = json.loads(data)
+
+            data = b''
+            while True:
+                data = data + self.client.recv(512 * 1024)
```

Comment:
People do in fact send more than 64kb of input, probably when they copy/paste the output.

This reads up to 512kb in a single chunk, and then if json load fails, which should only ever happen if it's incomplete, this tries to read more. In no case is this worse than previously, since even if it gets stuck in a forever-loop in some weird case, the alternative here would just be to crash.